automation/src/main/java/org/wso2/testgrid/automation/reader/JMeterTestReader.java

```diff
@@ -25,54 +25,55 @@
 import org.wso2.testgrid.common.TestScenario;
 
 import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import java.util.stream.Collectors;
 
 /**
  * Reader for reading Jmeter test files of the standard maven structure.
  */
 public class JMeterTestReader implements TestReader {
 
-    private static final String JMTER_SUFFIX = ".jmx";
     private static final String JMETER_TEST_PATH = "jmeter";
     private static final String SHELL_SUFFIX = ".sh";
     private static final String PRE_STRING = "pre-scenario-steps";
     private static final String POST_STRING = "post-scenario-steps";
+    private static final String SCENARIO_SCRIPT = "scenario.sh";
```

Comment:
Not required in this PR but shall we change this to read the pre, post script names and scenario script names to read from testgrid.yaml just like how we read deploy script and infra script?

Reply:
But the issue is there will be only one deploy script & infra script per test-plan and there will be multiple scenario scripts. So wouldn't it be hard to add the scenario script name in each test scenario rather than going to a convention?

Reply:
I guess we can use this name as the default. We can provide an option to over-ride this via testgrid.yaml if required. We should not dictate the script names of test writers. Our only requirement should be the .testgrid.yaml. :) 

WDYT?

Reply:
+1